.eslintrc.js

```diff
@@ -7,10 +7,15 @@ module.exports = {
   extends: [
     'eslint:recommended',
     'plugin:react/recommended',
-    'plugin:@typescript-eslint/recommended',
     'plugin:react-hooks/recommended',
     'plugin:jsx-a11y/recommended',
   ],
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      extends: ['plugin:@typescript-eslint/recommended'],
+    },
+  ],
```

Comment:
putting it here in the overrides, specific for '.ts' and '.tsx' files --> does that mean that ts-eslint only runs for '.ts' and '.tsx' files and not for all other files ('.js' etc.)? Was that the goal?